src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Service/CreateServiceCommandHandler.php

```diff
@@ -71,37 +68,17 @@ public function handle(CreateServiceCommand $command): void
         $service->setOrganizationNameEn($command->getOrganizationNameEn());
         $service->setOrganizationNameNl($command->getOrganizationNameNl());
         $this->serviceRepository->isUnique($service);
-        $this->serviceRepository->save($service);
-    }
-
-    private function createTeamData(string $name, string $teamName, string $email): array
-    {
-        $emails = $this->createEmailsArray($email);
 
-        return [
-            'name' => $teamName,
-            'description' => $this->translator->trans(
-                'teams.create.description',
-                [
-                '%teamName%' => $name,
-                ]
-            ),
-            'personalNote' => $this->translator->trans('teams.create.personalNote'),
-            'viewable' => false,
-            'emails' => $emails,
-            'roleOfCurrentUser' => 'MANAGER',
-            'invitationMessage' => $this->translator->trans('teams.create.invitationMessage'),
-            'language' => 'ENGLISH',
-        ];
-    }
+        $response = $this->inviteRepository->createRole(
+            "$serviceName {$command->getOrganizationNameEn()}",
+            "$serviceName {$command->getOrganizationNameEn()}",
```

Comment:
Suggestion: Use `sprintf` here, that is what is usually used in the project to create concatenations. Feel free to use this style, just givin' the heads up.